docs/labwc-config.5.scd

```diff
@@ -492,6 +492,17 @@ extending outward from the snapped edge.
 	If set to "no" (or is absent) the keybind will be layout agnostic.
 	Default is no.
 
+*<keyboard><keybind key="" onRelease="yes|no">*
+	*onRelease* makes keybind action take place on the release event of the
+	key. The most likely use-case for this is the binding of a Super key to a
+	a menu. Default: no. See example below:
```

Comment:
I have no idea how my comment disappeared, but I suggested something like:
```suggestion
	*onRelease*, when yes, fires the keybind action when the key or key combination is released, rather than first pressed. This is useful to bind actions to only modifier keys, when the action should fire when the modifier is pressed without another key. Default is no.
	
	The example below will trigger the launch of rofi when the super key is presed, without interfering with other multi-key combinations that include the super key:
```

Reply:
Thanks. Implemented.